features/bootstrap/FeatureContext.php

```diff
@@ -124,7 +124,7 @@ public function dump($arg1)
     public function iDontSee($arg1)
     {
         $element = $this->getSession()->getPage()->find('xpath', '//div[text()="' . $arg1 . '"]');
-        if (strpos('display: none', $element->getAttribute('style')) !== false) {
+        if (mb_strpos('display: none', $element->getAttribute('style')) !== false) {
```

Comment:
let's live with this... Even if `mb_strpos` should be never needed if both strings are utf-8 valid.